tests/test_date_field.py

```diff
@@ -0,0 +1,255 @@
+
+import pytest
+
+from werkzeug.datastructures import MultiDict
+from wtforms import Form
+from wtforms.validators import DataRequired, InputRequired
+
+from dmutils.forms.fields import DateField
+from dmutils.forms.validators import GreaterThan, Y2K
+
+import datetime
+
+
+class DateForm(Form):
+    date = DateField()
+
+
+def data_fromtuple(t, prefix='date'):
```

Comment:
Oh also we tend to underscore-prefix names that aren't meant for external use (appreciate this might not work with the fixtures below)